### PR TITLE
Ecf reports

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

**Comment by ericcf** [comment]
Related to: https://github.com/cbitstech/think_feel_do_engine/pull/57

File: spec/features/researchers/reports_spec.rb [hunk 1]
```diff
@@ -1,9 +1,14 @@
 require "rails_helper"
 
+class WozzleReport
```
Comment (gwensmu) [inline spec/features/researchers/reports_spec.rb]: Wozzle.

File: app/controllers/think_feel_do_dashboard/reports_controller.rb [hunk 2]
```diff
@@ -3,64 +3,29 @@
...
 
-    def reports
-      @reports = REPORTS
+    def fetch_reports
+      @reports = []
 
       return @reports unless Rails.application.config.respond_to?(:reports)
 
-      @reports += Rails.application.config.reports
+      @reports = Rails.application.config.reports
```
Comment (gwensmu) [inline app/controllers/think_feel_do_dashboard/reports_controller.rb]: So, the particular kinds of a reports available to an application will be set in a host app configuration as an array of report klasses?

Is this implemented yet in either of the host apps? I didn't see this in the commits.

Comment (ericcf) [inline app/controllers/think_feel_do_dashboard/reports_controller.rb]: See https://github.com/cbitstech/think_feel_do/blob/master/config/initializers/think_feel_do.rb


-------- Original message --------

**Comment by michaelwehrley** [comment]
Is this ready for merge?

**Comment by ericcf** [comment]
If you are both okay with it, I'll merge it when I merge the related tfd engine pull request. At that point, both host apps need updating as well.

**Pull request merged**